src/sonic-yang-models/sonic-acl.yang

```diff
@@ -0,0 +1,231 @@
+module sonic-acl {
+	namespace "http://github.com/Azure/sonic-acl";
+	prefix sacl;
+	yang-version 1.1;
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	import sonic-common {
+		prefix cmn;
+	}
+
+	import sonic-extension {
+		prefix sonic-ext;
```

Comment:
Like here, sonic- is used. but for sonic-port only port is used. This is little random. So lets use fixed patterns. I can correct then with MY PR, if needed.

Reply:
Sure go ahead Praveen